application/azuread_application.tf

```diff
@@ -122,3 +122,36 @@ resource "azuread_application" "app" {
     }
   }
 }
+
+resource "azuread_service_principal" "sp" {
+  count                         = var.azuread_service_principal == null ? 0 : 1
```

Comment:
Kan denne byttes til `for_each` slik at vi ikke får redeployment issues når/om en Enterprise Application slettes i listen?

Reply:
Dette vil vel ikke bli et problem da det kun er støtte for en SP per applikasjon. For_each vil vel også bare virke på maps, og om vi endrer typen til `map(object)` så vil det kunne se ut som om man kan lage flere SPer per applikasjon.